tools/picard.wdl

```diff
@@ -136,10 +136,17 @@ task bam_to_fastq {
     input {
         File bam
         String prefix = basename(bam, ".bam")
+        Boolean paired = true
         Int memory_gb = 40
         Int max_retries = 1
     }
 
+    parameter_meta {
+        bam: "Input BAM format file to convert to FastQ"
```

Comment:
```suggestion
        bam: "Input BAM format file to convert to FASTQ"
```

Reply:
@mcrusch - If I recall correctly, you have an opinion on this. Just wanted to make sure we invited your input before proceeding.

Reply:
We should use `FASTQ`.